packages/vue/src/composables/useAuth.ts

```diff
@@ -12,12 +12,15 @@ import { useClerkContext } from './useClerkContext';
  */
 function clerkLoaded(clerk: ShallowRef<Clerk | null>) {
   return new Promise<Clerk>(resolve => {
-    const unwatch = watch(clerk, value => {
-      if (value?.loaded) {
-        unwatch();
-        resolve(clerk.value!);
-      }
-    });
+    watch(
+      clerk,
+      value => {
+        if (value?.loaded) {
+          resolve(clerk.value!);
+        }
+      },
+      { immediate: true },
```

Comment:
We need to force the callback to be executed immediately to handle the case where clerk is already loaded when the watcher is created

Reply:
Also removed `unwatch` here since Vue will clean it up automatically when the component that uses `useAuth()` unmounts